### PR TITLE
docs: add skip consent recipe

### PR DESCRIPTION
**Pull request by big-kahuna-burger**
Adds example in recipes on how to skip consent

### CODE REVIEW

File: recipes/skip-consent.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+In order to accomodate those use-cases, you need to provide accepted property in interaction results whenever `interactionFinished` is called.
+
+```js
+const details = await provider.interactionDetails(req, res)
+// rest of your code...
+
+const result = {
+  select_account: {},
+  login: { account: account.accountId },
+  consent: { accepted: [details.params.scope.split(' ').filter(scope => WHITELISTED_SCOPES.includes(scope))] }
```
Comment (gigo1980) [inline recipes/skip-consent.md:36]: You have to add `accepted?: string[] | Set<string>` in the `types/index.d.ts`that this also works with typescript.

Comment (panva) [reply recipes/skip-consent.md:36]: Good. This line is actually incorrect. You don't pass what's `accepted`, you need to pass `rejectedScopes?: string[] | Set<string>` instead.

Comment (big-kahuna-burger) [reply recipes/skip-consent.md:36]: Will fix, thanks

File: recipes/skip-consent.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```js
+const details = await provider.interactionDetails(req, res)
+// rest of your code...
+
+const result = {
+  select_account: {},
+  login: { account: account.accountId },
+  consent: { 
+    rejectedScopes: ... // array of strings or Set of strings representing rejectedScopes
+    rejectedClaims: ... // array of strings or Set of strings representing rejectedClaims
```
Comment (panva) [inline recipes/skip-consent.md:38]:
```suggestion
    rejectedScopes: [], // array of strings representing rejected scopes, see below
    rejectedClaims: [], // array of strings representing rejected claims, see below
```

File: recipes/skip-consent.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Additionally, if you do remove consent prompt, you will get error when your RPs try to request scopes other than `openid` and `offline_access`.
+In order to accomodate those use-cases, you need to provide accepted property in interaction results whenever `interactionFinished` is called.
+
+```js
+const details = await provider.interactionDetails(req, res)
+// rest of your code...
+
+const result = {
+  select_account: {},
```
Comment (panva) [inline recipes/skip-consent.md:34]:
```suggestion
```

Remove this line.

File: recipes/skip-consent.md [hunk 4]
```diff
@@ -31,11 +31,10 @@ const details = await provider.interactionDetails(req, res)
 // rest of your code...
 
 const result = {
-  select_account: {},
   login: { account: account.accountId },
   consent: { 
-    rejectedScopes: ... // array of strings or Set of strings representing rejectedScopes
-    rejectedClaims: ... // array of strings or Set of strings representing rejectedClaims
+    rejectedScopes: [] // array of strings representing rejected scopes, see below
```
Comment (panva) [inline recipes/skip-consent.md]:
```suggestion
    rejectedScopes: [], // array of strings representing rejected scopes, see below
```

**Pull request merged**